run-in-docker.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -exo pipefail
+set -euo pipefail
```

Comment:
Although it looks like a small change, this would need a review from the core team, since it's not elixir specific

Could you separate this one out of the PR?

Reply:
@mrmstn good catch.

@halostatue What problem is this change going to solve?

Reply:
I can separate it out pretty easily. It’s a pair of minor changes, each of which could be separated.

1. `set -x` turns on tracing. With this, if you have:

   ```sh
   #!/usr/bin/env bash
   set -x
   echo Hello
   ```

   The output would be:

   ```console
   $ ./hello.sh
   + echo Hello
   Hello
   ```

   Removing the `x` from that cleans up the output.

2. `set -u` turns on errors for unset variables. Usually more useful on larger scripts, this would prevent the following from running (usually in error):

   ```sh
   #!/usr/bin/env bash
   echo $hello
   ```

   It’s fairly useful for catching variable name typos. (`echo $color` when your variable is `$colour`.)

I’m seeing a separate issue that should be addressed in this and other shell scripts, so I will pull it into its own PR anyway (hardcoding `#!/bin/bash` is an antipattern; the version of bash at `/bin`, if it exists, may not be the best version of bash to use on the system).